src/engine/runtime.js

```diff
@@ -457,6 +457,15 @@ class Runtime extends EventEmitter {
         return 'PERIPHERAL_SCAN_TIMEOUT';
     }
 
+    /**
+     * Event name for requesting an update of the indicator that shows that the
+     * microphone is being used to stream audio.
+     * @const {string}
+     */
+    static get UPDATE_MIC_INDICATOR () {
+        return 'UPDATE_MIC_INDICATOR';
```

Comment:
`UPDATE_MIC_INDICATOR: true/false` is a little ambiguous in what it should indicate (I think this name might also be too closely tied to the UI). I would suggest something like `MIC_LISTENING: true/false` or two events, `MIC_LISTENING_ON`/`MIC_LISTENING_OFF`.